README.md

```diff
@@ -1,27 +1,32 @@
-AncientWarfare2
+Ancient Warfare 2
 ===============
+
 In order to develop on this project you will need to:  
-Download the repository into a local folder.  
-Locate this folder, run gradle_update.bat  
-This should download all dependencies and setup proper .classpath  
-Import the project into eclipse from the downloaded folder/repository.  
+
+1) Download the repository into a local folder.  
+
+2) Locate this folder, run `gradlew.bat` or `./gradlew`.
+
+This should download all dependencies and setup proper `.classpath`. Import the project into Eclipse from the downloaded folder/repository.  
```

Comment:
And this should be either run gradlew eclipse or gradlew idea depending on which IDE you use.